biomage/utils/data.py

```diff
@@ -64,6 +64,10 @@ def copy_s3_files(sandbox_id, prefix, source_bucket, target_bucket):
     s3 = boto3.client("s3")
     exp_files = s3.list_objects_v2(Bucket=source_bucket, Prefix=prefix)
 
+    if 'Contents' not in exp_files:
+        click.echo(f"Failed to do an experiment copy: bucket {source_bucket} doesn't contain {prefix} as prefix.")
```

Comment:
mmm actually I think raising the exception might be better. If we are relying on experiments being copied for automated testing, if any of them is missing we should know instead of not testing that experiment.